test/EcdsaXi2023Cryptosuite.spec.js

```diff
@@ -0,0 +1,379 @@
+/*!
+ * Copyright (c) 2024 Digital Bazaar, Inc. All rights reserved.
+ */
+import {expect} from 'chai';
+
+import jsigs from 'jsonld-signatures';
+const {purposes: {AssertionProofPurpose}} = jsigs;
+
+import * as EcdsaMultikey from '@digitalbazaar/ecdsa-multikey';
+import {
+  credential,
+  ecdsaMultikeyKeyPair,
+  ecdsaSecp256KeyPair
+} from './mock-data.js';
+import {createCryptosuite} from '../lib/index.js';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+
+import {loader} from './documentLoader.js';
+
+const documentLoader = loader.build();
+const extraInformation = new Uint8Array([12, 52, 75, 63, 74, 85, 21, 5, 62, 10,
```

Comment:
Seems easier to read if aligned with the rest of the data.  Is that `10` supposed to be different than the `100`s below?
```suggestion
const extraInformation = new Uint8Array([
  12, 52, 75, 63, 74, 85, 21, 5, 62, 10,
```